enhancements/rhcos/split-rhcos-into-layers.md

```diff
@@ -0,0 +1,458 @@
+- Once all issues have been resolved, merge the PR. If new regressions are
+  encountered, revert the PR and fix them.
+- Once satisfied with the new bootimages, the RHCOS pipeline can stop building
+  the old bootimages containing OpenShift components.
+
+##### Phase 2: Switching `rhel-coreos` in CI
+
+- Set up a new job in openshift/release which builds the layered OpenShift node image on
+  top of the bootable container image from the new stream variant.
+- Open MCO and DTK PRs pointing to this layered image in CI (these are the only two components
+  that explicitly reference `rhel-coreos`) as a way to verify CI is happy.
+    - NOTE: There shouldn't be any major CI failures here. One key thing to understand is that
+      we're changing *how* the node image is built, but not *what* its final contents are from
+      a squashed image perspective. So this is mostly a sanity-check that they are indeed
+      equivalent.
+- Once any issues have been resolved, make the new job push to the `:rhel-coreos` tag of the
+  CI imagestream. This simultaneously requires disabling ART's job which syncs RHCOS to that
+  same tag.
+
+##### Phase 3: Switching `rhel-coreos` in production
+
+- Work with DPTP(?) to update the OCP release page to better handle the new layered approach.
+    - It currently links to diffs in the RHCOS release browser. But now the release browser
+      will only contain the RHEL diff, not the OCP diff in the node layer. We should close this
+      gap before moving prod.
+- Work with ART to set up a Konflux pipeline to build the layered OpenShift node image. 
+- Do a final set of tests to verify that CI is happy with this built image.
+    - There should be no surprises here.
+- Once any final issues have been resolved, switch over `rhel-coreos` in prod to come from
+  the node image built by the Konflux pipeline.
+
+### Follow-ups
+
+#### Re-organize repos
+
+With these changes in place, one follow-up worth investigating is to move the base RHCOS
+definition out of openshift/os. This would better clarify the dividing line between the CoreOS
+and node layers, by having their definitions live in separate repos.
+
+The CoreOS layer definition at this point would probably better live somewhere under gitlab.com/redhat
+alongside other rhel/centos-bootc projects. Then, openshift/os would be free to contain even more
+OpenShift-specific things. For example, we might consider moving content currently in the MCO's
+templates into openshift/os. or we might consider rewriting things currently written in bash
+into a statically-typed language that can literally live in openshift/os and get compiled and
+added as part of the derived (now multi-staged) layered build.
+
+At a more practical level, this helps reduce git churn for both the RHCOS pipeline and the
+OCP layered build.
+
+#### Add more comprehensive CI
+
+Another important follow-up would to be to hook up better CI testing to openshift/os. There is
```

Comment:
This is still possible even if ART continues to mirror rhcos into the CI imagestream. You would just disable ci-operator config "promotion" of the image. The openshift/os postsubmit job would then not push the CI image into the CI integration imagestream. 
ART would catch up with the changes later and push the latest updates to the CI integration imagestream.

Reply:
Right, makes sense.

Re. promotion, let's continue that discussion in the first conversation above on that topic.